test/libsolidity/StandardCompiler.cpp

```diff
@@ -1235,9 +1235,27 @@ BOOST_AUTO_TEST_CASE(use_stack_optimization)
 	BOOST_REQUIRE(contract["evm"]["bytecode"]["object"].isString());
 	BOOST_CHECK(contract["evm"]["bytecode"]["object"].asString().length() > 20);
 
-	// Now disable stack optimizations
+	// Now disable stack optimizations and UnusedFunctionParameterPruner
 	// results in "stack too deep"
 	parsedInput["settings"]["optimizer"]["details"]["yulDetails"]["stackAllocation"] = false;
+		parsedInput["settings"]["optimizer"]["details"]["yulDetails"]["optimizerSteps"] =
```

Comment:
I don't think copying this is a good idea. Can you instead look up the letter of the step in the list of steps and remove it from the string?